packages/next/src/lib/metadata/types/metadata-interface.ts

```diff
@@ -472,9 +476,18 @@ interface ResolvedMetadata extends DeprecatedMetadataFields {
   // if you provide an array it will be flattened into a single tag with comma separation
   keywords: null | Array<string>
   referrer: null | ReferrerEnum
+  /**
+   * @deprecated
+   */
   themeColor: null | ThemeColorDescriptor[]
+  /**
+   * @deprecated
+   */
   colorScheme: null | ColorSchemeEnum
-  viewport: null | string
+  /**
+   * @deprecated
+   */
+  viewport: null | Viewport
```

Comment:
I think these shoudl just be dropped

Reply:
Would need sometime for users to migrate, so deprecation annotation here is giving sometime to migrate but there're not working already. Will add some ts-doc here later for hints in editor